### PR TITLE
Mnd 7682 validate tokens

### PR DESCRIPTION
**Pull request by buenonadav**
- Implemented token validation in the Email OTP sample
- Updated READMEs
- Additional cosmetics changes

image

### CODE REVIEW

File: authentication-hub/package.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
   "description": "",
   "type": "module",
   "scripts": {
-    "start": "vite --debug --host --port 8080",
-    "start:local": "vite --debug --host --port 8082",
+    "start": "vite --debug --host",
```
Comment (ts-ron) [inline authentication-hub/package.json:7]: Why have you removed the port?

File: authentication-hub/vite.config.js [hunk 2]
```diff
@@ -11,4 +11,7 @@ export default defineConfig({
       handler: './backend/app.js',
     }),
   ],
+  server: {
+    port: process.env.PORT || 8080,
```
Comment (ts-ron) [inline authentication-hub/vite.config.js:15]: PLS speak with Yehonatan Stein - make sure he is not relying on a hard coded port we expose from the docker container

File: common.sh [hunk 3]
```diff
@@ -72,19 +72,21 @@ options=(
     "Authentication Hub"
     "Detection and Response Service"
     "Email OTP Authentication"
+    "SMS OTP Authentication"
+    "Token Validation"
```
Comment (ts-ron) [inline common.sh:76]: Hmm originally I tried an alphabetical sort - if you believe this is more convenient I am OK with it

**Review by ts-ron — APPROVED** [review]
LGTM, some questions but overall 🚀

File: authentication-hub/package.json (hunk 1 above)
Comment (buenonadav) [reply authentication-hub/package.json:7]: I recently moved all the samples to set the port using ENV var.
Simply aligned it here as well.

File: common.sh (hunk 3 above)
Comment (buenonadav) [reply common.sh:76]: I actually didn't go for alphabetical order.
I wanted the OTP related samples to be together.

File: authentication-hub/vite.config.js (hunk 2 above)
Comment (buenonadav) [reply authentication-hub/vite.config.js:15]: @ts-ron He's out sick today.
All the rest of our samples work the same way so I'm assuming there's no issue correct?

Comment (ts-ron) [reply authentication-hub/vite.config.js:15]: @buenonadav yes I spoke with him and its fine. So all 🟢 for merge

**Pull request merged**